mcenter_cli/parallelm/mcenter_cli/command_agent.py

```diff
@@ -0,0 +1,22 @@
+from cliff.command import Command
+
+
+class AgentRegisterCommand(Command):
+    def get_parser(self, prog_name):
```

Comment:
Shouldn't you be including Pydoc for your method defs? Example:

"""
       Blah, blah, blah...

        :param prog_name: Name of program.
           
        :return: 
        :raise:
        """

Reply:
All the classes inheriting from the Command are basically overriding the base class methods. So the doctoring for the get_parser and take_action would be the same all over..